Connectors/Tekla/Speckle.Connector.TeklaShared/Bindings/TeklaSelectionBinding.cs

```diff
@@ -1,49 +1,46 @@
 using Speckle.Connectors.DUI.Bindings;
 using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Eventing;
+using Speckle.Connectors.RhinoShared;
```

Comment:
Why do we importing from RhinoShared in Tekla?

Reply:
The events were declared in the wrong namespace but fixed in another PR: https://github.com/specklesystems/speckle-sharp-connectors/pull/505/commits/b5871d5fbda34a0e295971068eb3402a02971706